src/ui/Logo/Logo.ts

```diff
@@ -36,11 +36,12 @@ export class Logo extends Component {
     super(element, Logo.ID, bindings);
     this.options = ComponentOptions.initComponentOptions(element, Logo, options);
 
-    let link = $$(
+    const link = $$(
       'a',
       {
         className: 'coveo-powered-by coveo-footer-logo',
-        href: 'http://www.coveo.com/'
+        href: 'http://www.coveo.com/',
+        'aria-label': l('PoweredByCoveo')
```

Comment:
My intiution would be to label this element as `Coveo home page` so then a screen reader would read "A link with label Coveo home page". I don't think `PoweredByCoveo` is correct?